packages/firestore/test/util/test_platform.ts

```diff
@@ -272,11 +273,11 @@ export class TestPlatform implements Platform {
   }
 }
 
-/** Returns true if we are running under Node. */
-export function isNode(): boolean {
+/** Returns true if we are running in Safari. */
+export function isSafari(): boolean {
```

Comment:
Should it be moved to `@firebase/util`? It looks like something that should be shared.

Reply:
Moved.